features/step_definitions/basic_steps.rb

```diff
@@ -1,3 +1,20 @@
 Then("I should see {string}") do |content|
   expect(page).to have_content content
 end
+
+Then("{string} is present") do |selector|
+  expect(page).to have_selector :css, "#navigationBar"
+  expect(page).to have_selector :css, "#footer"
```

Comment:
2 expectations in one step checking for 2 differant (hard coded) selectors? Not sure about this...